Makefile

```diff
@@ -265,10 +265,14 @@ prebuild-package-deb-arm64: package-clean-deb-arm64
 
 build-package-deb-arm64:
 	@echo "Building Debian package..."
-# Prompt to edit changelog file
-	@nano $(CURRENT_DIR)/axolotl-$(AXOLOTL_VERSION)/debian/changelog
-# Prompt to edit copyright file
-	@nano $(CURRENT_DIR)/axolotl-$(AXOLOTL_VERSION)/debian/copyright
+# Edit changelog file
+	@sed -i.bak '4d' $(CURRENT_DIR)/axolotl-$(AXOLOTL_VERSION)/debian/changelog
```

Comment:
I dont have an arm apt system at hand to check, but I understand this creates a backup file - does it remove it later?
Thinking if this is ran with a CI, we will end up with plenty of .bak files?

Reply:
Thanks for reviewing and you are absolutely right! I forgot to remove `.bak` in this part, sorry!

Reply:
It's removed.